lib/Capabilities.php

```diff
@@ -0,0 +1,126 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\PasswordPolicy;
+
+use OCP\Capabilities\ICapability;
+
+/**
+ * Class Capabilities
+ *
+ * @package OCA\PasswordPolicy
+ */
+class Capabilities implements ICapability {
+
+	/** @var ConfigProvider */
+	private $configProvider;
+
+	/**
+	 * Capabilities constructor.
+	 *
+	 * @param ConfigProvider $configProvider
+	 */
+	public function __construct(ConfigProvider $configProvider) {
+		$this->configProvider = $configProvider;
+	}
+
+	/**
+	 * Return this classes capabilities
+	 *
+	 * array['password_policy']              		array Defines the capabilities
+	 *          [password_requirements]      		array Defines the password requirements
+	 *          	['minimum_characters']			integer (optional) Defines the minimum password length
+	 *              ['configuration']         		array Defines the password requirements base configuration (for e.g password managers)
+	 *              	['lower_case']       		array Defines the password requirements lower case configuration
+	 *              		['generate_from']       array Defines the possible password parameters
+	 *              			['characters']		string Defines the allowed characters (default abcdefghijklmnopqrstuvwxyz)
+	 *              		['minimum']       		integer (optional) Defines the minimum amount of lower case characters
+	 *              	['upper_case']       		array Defines the password requirements upper case configuration
+	 *              		['generate_from']       array Defines the possible password parameters
+	 *	            			['characters']      string Defines the allowed characters (default ABCDEFGHIJKLMNOPQRSTUVWXYZ)
+	 *              		['minimum']       		integer (optional) Defines the minimum amount of upper case characters
+	 *              	['numbers']       			array Defines the password requirements numbers configuration
+	 *              		['generate_from']       array Defines the possible password parameters
+	 *          				['characters']      string Defines the allowed characters (default 1234567890)
+	 *              		['minimum']       		integer (optional) Defines the minimum amount of numbers
+	 *              	['special_characters']      array Defines the password requirements special characters configuration
+	 *              		['generate_from']       array Defines the possible password parameters
+	 *           				['characters']      string (optional) Defines the allowed characters
+	 * 							['any]				boolean (optional) Defines whether all special characters are allowed (default true)
```

Comment:
seems too indented

Reply:
Thought so as well, but if you look in any editor on your local machine, it looks good